mdbenchmark/plot.py

```diff
@@ -147,35 +147,50 @@ def filter_dataframe_for_plotting(df, host_name, module_name, gpu, cpu):
 
 @cli.command()
 @click.option("--csv", help="Name of CSV file to plot.", multiple=True)
-@click.option("--output-name", "-o", help="Name of output file.")
+@click.option("-o", "--output-name", help="Name of output file.")
 @click.option(
     "--output-type",
-    "-t",
     help="File extension for output file.",
     type=click.Choice(["png", "pdf", "svg", "ps"]),
     show_default=True,
     default="png",
 )
 @click.option(
-    "--module-name",
     "-m",
+    "--module",
+    "module",
     multiple=True,
     help="Name of the MD engine module(s) to plot.",
 )
-@click.option("--host-name", "-h", multiple=True, help="Name of hosts to plot.")
 @click.option(
-    "--gpu/--no-gpu", help="Plot data of GPU runs.", show_default=True, default=True
+    "-t",
+    "--template",
+    "--host",
+    "template",
```

Comment:
I know we discussed this but I am wondering whether it is a good idea to use two names here. It might be confusing what is meant... i think we should settle one name

Reply:
So I was thinking of starting to deprecate `--host` and advertise `--template` as the one default. In the current implementation `--host` will be listed last.